README.rst

```diff
@@ -161,7 +161,13 @@ and allows to define the geographic coordinates of the devices,
 as well as their indoor coordinates on floorplan images.
 
 This module also provides an API through which mobile devices can update
-their coordinates.
+their coordinates. See below for further details:
+
+.. code-block:: text
```

Comment:
used this method because I am trying to do something similar to what was done here https://github.com/openwisp/openwisp-firmware-upgrader#rest-api